cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinTitle.java

```diff
@@ -15,7 +15,8 @@
     public static final String MAIN_TITLE = "titteltekst";
     public static final String LANGUAGE_CODE = "sprakkode";
     public static final String ABSTRACT = "abstract";
-    public static final String VALUE_OF_ORIGINAL_TITLE_STATUS = "J"; //probably means "Ja"
+    public static final String ORIGINAL_TITLE = "J"; //probably means "Ja"
+    public static final String NOT_ORIGINAL_TITLE = "N"; //probably means "Ja"
```

Comment:
I guess that "N" probably has a misleading comment. I guess that I might have called these something else, but I don't think it is worth changing